herddb-services/src/main/resources/conf/server.properties

```diff
@@ -61,12 +61,21 @@ server.ssl=false
 # Enable/Disable network acceptor
 # server.network.enabled=true
 
-# for server.mode=cluster
+# for server.mode=cluster you have to set a connection string to your ZooKeeper cluster
+# you have to write here the list of all of the zookeeper servers, not only one
 server.zookeeper.address=localhost:2181
 server.zookeeper.session.timeout=40000
 server.zookeeper.path=/herd
 
+# data replication settings
+# this is the default value for 'expectedreplicacount' for CREATE TABLESPACE command
```

Comment:
Based on the comment here it seems that the replica count can and should be less than or equal to the number of nodes in the cluster. Are we checking for that somewhere or throw an error if the user accidentally chooses a number that is higher?

Reply:
This is somehow tricky to understand.
We are setting a default value for expectedreplicacount that is a "target replication" factor for data.
In order to create a tablespace all you need is only a *leader* node.
When you set *expectedreplicacount* the system will try to have that number of replicas *eventually*.
It is a limit that it is not enforced.

We have to parameters for replication:
-bookkeeper write replication (writequorumsize)
- data replication

Reply:
I see. Thanks for explaining this.